internal/local/tool.go

```diff
@@ -33,38 +32,38 @@ const (
 	// The gke-local deployer spawns one controller and one distributor/manager
 	// per region. The controller and distributors all listen on different
 	// ports. The store is used to assign ports to regions.
-	proxyPort      = 8000                 // proxy port
-	controllerPort = 8001                 // controller port
-	portsKey       = "/distributor_ports" // mapping from region to distributor port
-	projectName    = "local"              // fake name for the local GCP "project"
+	proxyPort      = 8000           // proxy port
```

Comment:
This will be the port that the application listeners will be reachable on. I think we need to pick a port that's well known and expected for the user. Do you have any idea for a better port?

Reply:
Oh yeah, I overlooked that! Maybe let's just leave it as is then.